src/migrations/20231030150724-sim_score_cache.js

```diff
@@ -0,0 +1,55 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      const sessionSig = __filename;
+      await prepMigration(queryInterface, transaction, sessionSig);
+      await queryInterface.createTable(
+        'SimScoreCaches',
```

Comment:
I think this needs to have goal in the name of the table, we may need to have similarity scores for other entity types in the future. Objectives, Uploaded resources, ...

Reply:
be077c98b3d91e2d120df36d2b124810edf217d8